server/models/girl.js

```diff
@@ -0,0 +1,46 @@
+const Sequelize = require('sequelize')
+const sequelize = require('./connection');
+
+module.exports =  sequelize.define('girl', {
+    id: {
+      type: Sequelize.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true
+    },
+    user_name: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    first_name: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    last_name: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    email: {
+      type: Sequelize.STRING,
+      allowNull: false
+    },
+    school_id: {
```

Comment:
What happens in the app if a girl attended a school that's not (yet) in the database?  Would you let them enter the school name as text and then add it to the schools table, or would you not have a school_id set for that girl? Or something else?

Reply:
If there an API for UK schools to check the school firstly it will be good to solve this problem, but if no, the name of the school very important because it will help in the next  -school phase-

Reply:
I added "other_school" field to the girl table with "STRING" type to be filled in case the school was not listed (and the school_id is nullable).
As you said, this field could be grouped periodically so to update the school table.

Reply:
Cool, that sounds like a good solution.